src/UI.elm

```diff
@@ -9,7 +9,7 @@ import Board exposing (Board, Mark(..), toArray)
 import Array exposing (..)
 import Game exposing (Game, GameState(..), retrieveState, board, currentPlayer)
 
-type Msg = TakeTurn Int | Reset
+type Msg = TakeTurn Int | Reset | GameType String
```

Comment:
I'd prefer the game type to be a proper type because there's a small list of them and they don't need to be very flexible. Pseudocode:

```elm
type GameType = HvH | HvC
type Msg = TakeTurn Int | Reset | SelectGameType GameType
```